### ISSUE TITLE
View your works accessibility audit 

### ISSUE THREAD

**Issue by kellyChex** [samvera/hyrax#3966]
### Descriptive summary

In testing for the Hyrax 3.0 release, the following points are actions that need to be taken on the Hyrax view your works page in order to be more accessible:

**ARIA hidden element must not contain focusable elements:**
![Screen Shot 2019-09-11 at 5 19 37 PM](https://user-images.githubusercontent.com/9621070/64744365-5d8b6c00-d4b8-11e9-8de3-826c5d3dc6e1.png)

![Screen Shot 2019-09-11 at 5 20 05 PM](https://user-images.githubusercontent.com/9621070/64744381-6e3be200-d4b8-11e9-961f-3627cbcdc1af.png)

![Screen Shot 2019-09-11 at 5 20 32 PM](https://user-images.githubusercontent.com/9621070/64744393-7f84ee80-d4b8-11e9-917c-9db2bb03c7d4.png)

![Screen Shot 2019-09-11 at 5 24 39 PM](https://user-images.githubusercontent.com/9621070/64744526-105bca00-d4b9-11e9-856a-1adf87eba765.png)

![Screen Shot 2019-09-11 at 5 25 07 PM](https://user-images.githubusercontent.com/9621070/64744538-1f427c80-d4b9-11e9-9779-9b73547f011c.png)

![Screen Shot 2019-09-11 at 5 25 32 PM](https://user-images.githubusercontent.com/9621070/64744567-2ec1c580-d4b9-11e9-8926-0c162ac70496.png)

![Screen Shot 2019-09-11 at 5 25 58 PM](https://user-images.githubusercontent.com/9621070/64744576-3da87800-d4b9-11e9-8cb5-9b62b568527c.png)

![Screen Shot 2019-09-11 at 5 26 20 PM](https://user-images.githubusercontent.com/9621070/64744587-4bf69400-d4b9-11e9-9d9b-600bf278d6c6.png)

![Screen Shot 2019-09-11 at 5 26 45 PM](https://user-images.githubusercontent.com/9621070/64744612-59ac1980-d4b9-11e9-8de5-eda36e64ac2c.png)

![Screen Shot 2019-09-11 at 5 27 07 PM](https://user-images.githubusercontent.com/9621070/64744629-692b6280-d4b9-11e9-9c18-4a2043165aa5.png)

**`<li>` elements must be contained in a `<ul> `or `<ol>`**

![Screen Shot 2019-09-11 at 5 27 57 PM](https://user-images.githubusercontent.com/9621070/64744660-84966d80-d4b9-11e9-97b8-b2cb83a739ca.png)

![Screen Shot 2019-09-11 at 5 28 19 PM](https://user-images.githubusercontent.com/9621070/64744681-9710a700-d4b9-11e9-85c4-0addb0208d82.png)

**ARIA role must be appropriate for the element:**
![Screen Shot 2019-09-11 at 5 29 03 PM](https://user-images.githubusercontent.com/9621070/64744702-ad1e6780-d4b9-11e9-95e8-c3aaf0296642.png)

![Screen Shot 2019-09-11 at 5 29 28 PM](https://user-images.githubusercontent.com/9621070/64744714-bc051a00-d4b9-11e9-8dca-d48f530acfcf.png)

**Checkbox inputs with the same name attribute value must be part of a group:**
![Screen Shot 2019-09-11 at 5 31 37 PM](https://user-images.githubusercontent.com/9621070/64744776-0a1a1d80-d4ba-11e9-959b-3ed1c6ff3c95.png)

**Ensures landmarks are unique:**
![Screen Shot 2019-09-11 at 5 32 14 PM](https://user-images.githubusercontent.com/9621070/64744796-1dc58400-d4ba-11e9-982c-c3df21109b6a.png)

### Steps to reproduce the behavior

1. Install the Axe plugin or addon from Deque (https://addons.mozilla.org/en-US/firefox/addon/axe-devtools/ is the firefox version).
2. Open https://nurax-dev.curationexperts.com/dashboard/my/works?locale=en
3. Open the inspector (right-click, and select "Inspect element")
4. Click on the 'Axe' tab
5. Click 'Analyze'

### Tab Test 

In Firefox and Safari, unable to tab through many of the links such as the links to the works. 

Chrome works well. 

![Screen Recording 2019-09-16 at 01 57 PM](https://user-images.githubusercontent.com/9621070/64993069-0efc1a00-d88a-11e9-9daf-7eb53be0d8d9.gif)

**Comment by dlim87** [comment]
Axe found 4 errors, 2 major ones for this page:
## Images must have alternate text
![Image 2019-11-18 at 7 51 35 AM](https://user-images.githubusercontent.com/5492162/69067651-48f7b280-09d8-11ea-88d5-bffbc70b04e5.png)
## Heading levels should only increase by one
![Image 2019-11-18 at 7 52 11 AM](https://user-images.githubusercontent.com/5492162/69067684-6167cd00-09d8-11ea-8c1c-8bb090d67ada.png)

And 2 minor ones that we decided were probably just due to Nurax's implementation instead of hyrax:
## All page content must be enclosed by landmarks
![Image 2019-11-18 at 7 54 37 AM](https://user-images.githubusercontent.com/5492162/69067869-aee43a00-09d8-11ea-93e7-483eeb64a01f.png)
## Elements must have sufficient color contrast
![Image 2019-11-18 at 7 54 56 AM](https://user-images.githubusercontent.com/5492162/69067911-c3283700-09d8-11ea-8878-e7c7aeac256f.png)

**Comment by dlim87** [comment]
@Dananji ^

**Comment by MariaStarPower** [comment]
This **passes**, but with some minor issues.  Also tabbing with Firefox is limited, as I cannot tab through everything on the page.

Results of Axe analysis of the view your works page:

<details>

![image](https://user-images.githubusercontent.com/32848057/101067179-13265d00-354c-11eb-96a8-927316fab8c6.png)

![image](https://user-images.githubusercontent.com/32848057/101067243-2afde100-354c-11eb-91a1-890f8cfe6329.png)

![image](https://user-images.githubusercontent.com/32848057/101067323-44069200-354c-11eb-8360-62ffde51ba41.png)

![image](https://user-images.githubusercontent.com/32848057/101067388-58e32580-354c-11eb-8061-8c0e0d751034.png)

</details>

**Comment by crisr15** [comment]
I am pushing this back for someone to look at the tabbing on Firefox and decide if it needs to be fixed, or if this passes.

**Comment by jlhardes** [comment]
This issue seems to be a report of an accessibility audit and might point to multiple things that need to be fixed. Review needed to determine work that remains to be done. Not a blocker for Hyrax 3.0 release.

**Comment by rjkati** [comment]
Per conversation in hyrax-wg slack, I created a new ticket for the issues that @MariaStarPower flagged: https://github.com/samvera/hyrax/issues/4753
